### ISSUE TITLE
Consider adding Package Icon

### ISSUE THREAD

**Issue by bnice5000** [OCJvanDijk/Brightness-Sync#7]
The package doesn't have its own icon. Consider adding an icon to the package so that it does not just have the generic package icon in applications.

**Comment by OCJvanDijk** [comment]
I would want to, but I don't have the skills to make a high resolution icon that would be better than no icon. Actually I'm not even sure it's worth it, as you don't generally see it anyway.

A menu bar icon is a lot easier and you actually see that most of the time, so I tried to make something acceptable. But for the app icon, I figured you could just bury the app somewhere deep in your Applications folder so you don't have to look at it (which is what I do).

If anyone else feels like making an icon I would be happy to include it of course.

**Comment by bnice5000** [comment]
Alright, si I had to play around with this a little. I am not an artist either. So I started with an [open source SVG](https://publicdomainvectors.org/en/free-clipart/Sun-vector-image/10956.html). A sun seemed fitting, so I went with it. I then saved the SVC as a 1024x1024 png. Then I create a script to turn the png into an icns.

```
mkdir Sun.iconset
sips -z 16 16     Sun.png --out Sun.iconset/icon_16x16.png
sips -z 32 32     Sun.png --out Sun.iconset/icon_16x16@2x.png
sips -z 32 32     Sun.png --out Sun.iconset/icon_32x32.png
sips -z 64 64     Sun.png --out Sun.iconset/icon_32x32@2x.png
sips -z 128 128   Sun.png --out Sun.iconset/icon_128x128.png
sips -z 256 256   Sun.png --out Sun.iconset/icon_128x128@2x.png
sips -z 256 256   Sun.png --out Sun.iconset/icon_256x256.png
sips -z 512 512   Sun.png --out Sun.iconset/icon_256x256@2x.png
sips -z 512 512   Sun.png --out Sun.iconset/icon_512x512.png
cp Sun.png Sun.iconset/icon_512x512@2x.png
iconutil -c icns Sun.iconset
rm -R Sun.iconset
```
I am not exactly sure how to link it in your binary pkg yet. I am not really an xcode/Objective-C programmers so this is all a little out of my wheelhouse. 

If you have an idea, I am happy to try.

[icon.sh.txt](https://github.com/OCJvanDijk/Brightness-Sync/files/4415089/icon.sh.txt)
![Sun](https://user-images.githubusercontent.com/100688/78137992-8197a600-7426-11ea-9e22-9f60538e5fc8.png)

**Comment by OCJvanDijk** [comment]
Hey, thanks for your help. Good idea to use public domain vectors, didn't think of that.

So now you got me spending time in Pixelmator anyway, while I should be doing more important things 😅. I appreciate having a fun way to procrastinate though. I used your icon as a starting point and added a sync icon in the middle, to make it clear what it's about.

Including a 1024x1024 icon increases the app size 7x, from 167kb to 1.2mb, which is... fine I guess?

I've realized this before about the menu bar icon, but the two arrows don't actually make sense, because the app only syncs one way, ah well.

Btw, you only need one 1024x1024 png for your app icon. Unless you are going to optimize your icon for every size, there is no reason to generate each size yourself, macOS will just scale your icon down.

Thoughts?

![Brightness Sync 1024](https://user-images.githubusercontent.com/7405518/78240572-7ce2f880-74df-11ea-8c81-8565d347e1a1.png)
